dhctl/pkg/config/base.go

```diff
@@ -289,12 +361,21 @@ func ParseConfigFromData(ctx context.Context, configData string, preparatorProvi
 
 	// init configuration can be empty, but we need default from openapi spec
 	if len(metaConfig.InitClusterConfig) == 0 {
+		// If we have multiple documents but InitConfiguration wasn't found, it might be due to missing separator
```

Comment:
hm, we merge this pr https://github.com/deckhouse/deckhouse/pull/16103/changes with new registry behavior  we can use ModuleConfig deckhouse for init configuration instead of init configuration.

And i think this validation can be deleted. But please connect with regestry command for check it